tests/metrics_test.py

```diff
@@ -128,3 +130,35 @@ def test_c2st_scores(dist_sigma, c2st_lowerbound, c2st_upperbound):
     assert obs2_c2st.mean() <= c2st_upperbound
 
     assert np.allclose(obs2_c2st, obs_c2st, atol=0.05)
+
+
+@pytest.mark.slow
```

Comment:
could you add tests for the other distances as well? that'd be great!

Reply:
sure, I'll add them :)

Reply:
Added tests for the unbiased and biased MMD based on hypothesis tests.